website/loaders/redirectsFromCsv.ts

```diff
@@ -45,45 +56,39 @@ const getRedirectFromFile = async (
     .slice(1)
     .map((row) => {
       // this regex is necessary to handle csv with comma as part of value
-      const parts = row.split(/,(?=(?:(?:[^"]*"){2})*[^"]*$)/);
-
-      const typeRowIndex = parts.findIndex((part: string) =>
-        REDIRECT_TYPE_ENUM.includes(part)
-      );
+      const parts = row.split(/[,;](?=(?:(?:[^"]*"){2})*[^"]*$)/);
```

Comment:
avoid new separators. it may break current users.

```suggestion
      const parts = row.split(/,(?=(?:(?:[^"]*"){2})*[^"]*$)/);
```